### ISSUE TITLE
Keyboard focus indicator is not visible when focus moves to the single overflow item

### ISSUE THREAD

**Issue by linazar** [microsoft/fluentui#20664]
### Environment Information

- **Package version(s)**: Fluent React 8.42.3
- **Browser and OS versions**: Edge 95.0.1020.44 ; Windows 11 Enterprise

### Describe the issue:

Keyboard focus indicator is not visible when focus moves to the single overflow item. It is visible with multiple items, but not when there is just one. 

#### Actual behavior:

Keyboard focus indicator is not visible when focus moves to the single overflow item. It is visible with multiple items, but not when there is just one. 

#### Expected behavior:

Ensure tabs in overflow menu have unique id attributes

### If applicable, please provide a codepen repro:

1) Navigate to https://developer.microsoft.com/en-us/fluentui?fabricVer=8#/controls/web/pivot, make sure 8.42.3 is selected

2) Via keyboard (Tab/Tab+shift and/or arrow keys) navigate to "Pivot with overflow menu"

3) Focus is not visible around the "The Last Tab"

4) Zoom the page so that the overflow menu contains more items - the focus is visible, although it seems to run onto other options.

5) According to accessibility assessment team, there should be focus around the menu item even if it is a single item. See this bug for screenshot https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/8030887

**Comment by gouttierre** [comment]
@linazar - Thanks for filing this issue with us. To set expectations, the developers will review this issue once capacity allows.

@spmonahan - Would you be able to confirm if this is a regression, or if this behavior is an issue? 
cc @smhigley

**Comment by ElinorW** [comment]
Hi @gouttierre,

An issue close to this, at a lower resolution (1280 x 768), tabbing through the elements causes some text to overlay the menu items and thus the menu items can't be read
![image](https://user-images.githubusercontent.com/18407044/150947062-66db96c1-53c5-4d01-bcdf-11e5e3ac37cd.png)

**Comment by micahgodbolt** [comment]
I figured out why the ghost text is showing up:
pseudo element with data-content as the content

![image](https://user-images.githubusercontent.com/1434956/163238413-5a13ed56-4f81-4955-abf0-3ef41e1ebe61.png)

Now to figure out why this CSS was added in the first place, and what its intention was.
